packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -342,38 +372,67 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
           {isOpen && (
             <ScrollContainer showScrollIndicators maxHeight="inherit" ref={scrollRef} padding={0.5}>
               {!asyncError && (
-                <ul style={{ height: rowVirtualizer.getTotalSize() }} className={styles.menuUlContainer}>
-                  {rowVirtualizer.getVirtualItems().map((virtualRow) => {
+                <div style={{ height: rowVirtualizer.getTotalSize() }} className={styles.menuUlContainer}>
```

Comment:
Wonder why this switch to divs, isn't it worse for the a11y?

Reply:
Not really - we're preserving the HTML roles of `menulist` and `item` on the correct elements, so the ul/li didn't really give us much in the first place, and then became less correct once we started adding in group headers insite the `<li>`s